osu.Desktop/DiscordRichPresence.cs

```diff
@@ -135,6 +135,9 @@ private string getDetails(UserActivity activity)
 
                 case UserActivity.Editing edit:
                     return edit.Beatmap.ToString();
+
+                case UserActivity.InLobby lobby:
+                    return lobby.Room.Name.ToString();
```

Comment:
weird to call `.ToString()` on a... string - the rest have `.ToString()` calls for better formatting, but this one doesn't make much sense at all